git-glossary.md

```diff
@@ -13,6 +13,11 @@ first 7 characters of the 40-character string.
 
 As a verb: The action of storing a new snapshot of the project's state in the Git history, by creating a new commit. 
 
+## fork
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without
```

Comment:
Can we add here that a fork is stored on github and that it maintains a connection to the original repo? I've spent 30 minutes reading about the difference between clones and forks, trying to wrap my head around it. The connection to the original repo and that they are stored on github seem to stick out to me, but i'm a newbie in this area

Reply:
I amended the definition of fork. Please see if it is what you had in mind.